src/Input/Select/Select.tsx

```diff
@@ -24,82 +24,118 @@ const Select: ISelect = (props: Props) => {
     onFocus,
     onBlur,
     onChange,
+    onInputChange,
     noOptionResult = 'No results found',
     small,
     disableTyping,
     removeFloatingLabel,
     removeDropIcon,
     error,
     renderError,
-    value, // eslint-disable-line @typescript-eslint/no-unused-vars
-    children, // eslint-disable-line @typescript-eslint/no-unused-vars
+    value,
+    defaultValue,
```

Comment:
`defaultValue` should be passed to `SelectInput`

Reply:
it's being used to set `inputValue` already.